### PR TITLE
Improve the formatting.

### PR DESCRIPTION
**Pull request by malukenho**
Use classes for tables and replace `ul/li` by `dt/dd`.

Insert a few spaces for better viewing of the tables after rendered.

### CODE REVIEW

File: source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html [hunk 1]
```diff
@@ -144,17 +144,18 @@
     As you can see, we are now joining 2 tables in the results, and the ORM has to perform more complicated operations:
 </p>
 
-<ol>
-    <li>
-        Hydrate
-        <strong>1</strong> 
-        <code>User</code> 
-        object for
+<dl class="dl-horizontal">
```
Comment (Ocramius) [inline source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html]: Hmm... Semantically, this is a list, not a definition list...

File: source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html [hunk 2]
```diff
@@ -290,7 +296,7 @@
             <td>21</td>
             <td>@ocramius</td>
             <td>Twitter</td>
-            <td>b1fc7171/ocramius-macbook</td>
+            <td>b1fc7171/ ocramius-macbook</td>
```
Comment (Ocramius) [inline source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html]: These spaces are not supposed to be there (assuming session ID contains device ID)

File: source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html (hunk 1 above)
Comment (malukenho) [inline source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html]: sounds like a **steps definition** for me

File: source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html (hunk 2 above)
Comment (malukenho) [inline source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html]: It's needed to table don't broken grid.

Comment (Ocramius) [inline source/_posts/2015-04-10-doctrine-orm-optimization-hydration.html]: I see - will try to simply remove the hash then

**Pull request merged**